### DISCUSSION TITLE
(Parte 1) Hashing

### DISCUSSION THREAD

**Discussion by CarlangaUC** [IIC2133-PUC/2024-1#148] [Tarea 2] [answered]
Hola!

Venia a consultar por el hecho de que recomiendan emplear rolling hashing mas especificamente tiran el hint de Rabin-Karp, esta bien si ocupamos otras estrategias de comparacion de caracteres?

**Comment by sebacarrascop** [comment, marked as answer]
hola!, mientras cumplas con la complejidad pedida no hay problema!
